test/parse.spec.js

```diff
@@ -36,6 +35,24 @@ describe('readSocketConfig()', () => {
     })
   })
 
+  it('should read and parse socket.yml v1', async () => {
+    const fileContent = await readFile(path.resolve(__dirname, 'sample-v1.yml'), 'utf8')
+
+    await parseSocketConfig(fileContent).should.eventually.become({
+      'githubApp': {
+        'enabled': true,
+        'projectReportsEnabled': false,
+        'pullRequestAlertsEnabled': true,
+      },
+      issueRules: {},
```

Comment:
```suggestion
      'issueRules': {},
```

Minor nitpick for consistency